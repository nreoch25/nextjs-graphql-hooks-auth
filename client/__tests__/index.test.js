import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import IndexPage from "../pages/index";

describe("Index Page", () => {
  it("matches the snapshot", () => {
    const wrapper = shallow(<IndexPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
