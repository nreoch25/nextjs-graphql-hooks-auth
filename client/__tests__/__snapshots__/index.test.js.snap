// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page matches the snapshot 1`] = `
<Layout>
  <Index />
</Layout>
`;
